core/shared/src/main/scala/zio/FiberId.scala

```diff
@@ -76,7 +76,7 @@ object FiberId {
 
   private[zio] val _fiberCounter = new java.util.concurrent.atomic.AtomicInteger(0)
 
-  case object None                                           extends FiberId
+  final case object None                                     extends FiberId
```

Comment:
I think this will result in an error on Scala 3.

Reply:
Seems like it passes :crossed_fingers: